apps/hubble/src/storage/db/types.ts

```diff
@@ -101,6 +101,10 @@ export enum UserPostfix {
   UsernameProofMessage = 7,
 
   // Add new message types here
+
+  /* Tag */
```

Comment:
Note that this needs to be exactly duplicated in the `message.rs` file in the Rust addon